compiler/luci/export/src/CircleTensorExporter.cpp

```diff
@@ -559,6 +559,8 @@ bool has_same_values(luci::CircleConst *lhs, luci::CircleConst *rhs)
       return has_same_elements<loco::DataType::S4>(lhs, rhs);
 
     case loco::DataType::S8:
+      if (lhs->size<loco::DataType::S8>() != rhs->size<loco::DataType::S8>())
+        return false;
```

Comment:
Q) why can the size be different?

Reply:
> Q) why can the size be different?

In case when weights are compressed we don't change shape information, so size != WxHxDxL and we need additional check that sizes are equal before comparison